src/common/utils/flipCard.js

```diff
@@ -0,0 +1,8 @@
+export const flipCard = card => {
+  const newCard = {
```

Comment:
If you want to copy an object, but change a few properties, here's a common pattern:

```
const newCard = { ...card, isFaceUp: !card.isFaceUp }
```

`...card` copies the existing properties of `card`.

There are two main benefits to this approach:
1. Avoid manually copying properties
2. Make sure you copy all properties

Reply:
Ooh, I like this!